### PR TITLE
Refs #23882 - fix lint errors and translations

### PR DESCRIPTION
**Pull request by tstrachota**
- type property was missing in RepositorySetRepositories
- `<translate>` directive from Angular doesn't work in React

### CODE REVIEW

**Comment by akofink** [comment]
I don't quite understand. I've been running `npm run lint` to lint locally. Here's what I get on master:

```
$ npm run lint

> TheForemanDevDeps@1.20.0 lint /home/vagrant/nfs/foreman
> eslint -c .eslintrc webpack/ script/ || exit 0


/home/vagrant/nfs/foreman/webpack/assets/javascripts/compute_resource/ovirt.js
  103:3   error  Identifier 'size_gb' is not in camel case            camelcase
  103:12  error  Identifier 'storage_domain' is not in camel case     camelcase
  103:42  error  Identifier 'disk_interface' is not in camel case     camelcase
  103:58  error  Identifier 'wipe_after_delete' is not in camel case  camelcase

✖ 4 problems (4 errors, 0 warnings)


> TheForemanDevDeps@1.20.0 postlint /home/vagrant/nfs/foreman
> node ./script/npm_lint_plugins.js


> katello@1.0.0 lint /home/vagrant/nfs/katello
> eslint webpack/


> foreman_remote_execution@1.0.0 lint /home/vagrant/.rvm/gems/ruby-2.4.4/gems/foreman_remote_execution-1.6.1                                                                                                      
> eslint -c .eslintrc webpack/ script/ || exit 0
```

How should I be linting?

File: webpack/scenes/RedHatRepositories/components/RepositorySetRepository.js [hunk 1]
```diff
@@ -69,18 +69,22 @@ class RepositorySetRepository extends Component {
...
-      <span translate>
-        This repository is not suggested. Please see additional&nbsp;
-        <a href="https://access.redhat.com/articles/1586183">documentation</a>
-        &nbsp;prior to use.
-      </span>
-    );
+
+    const yStreamHelpText =
+      sprintf(
+        __('This repository is not suggested. Please see additional %(anchorBegin)s documentation %(anchorEnd)s prior to use.'),
```
Comment (akofink) [inline webpack/scenes/RedHatRepositories/components/RepositorySetRepository.js]: nit: the link is rendered as "documentation " rather than "documentation"

**Comment by johnpmitsch** [comment]
@tstrachota Is eslint not working in jenkins?  I thought it had been fixed :/

**Comment by tstrachota** [comment]
I fixed spaces around "documentation".

When I run eslint on master locally I get:
```
npm run lint

> katello@1.0.0 lint /home/vagrant/foreman/katello
> eslint webpack/


/home/vagrant/foreman/katello/webpack/scenes/RedHatRepositories/components/RepositorySetRepositories.js
  20:19  error  'type' is missing in props validation  react/prop-types

✖ 1 problem (1 error, 0 warnings)
```
The same when I execute it from the foreman. I have no idea what is causing the differences :-/

**Comment by akofink** [comment]
ACK!

**Pull request merged**